java-package/mxnet-engine/src/main/java/org/apache/mxnet/training/ParameterServer.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mxnet.training;
+
+import org.apache.mxnet.ndarray.NDArray;
+
+import java.util.Arrays;
+
+/** An interface for a key-value store to store parameters, and their corresponding gradients. */
+public interface ParameterServer extends AutoCloseable {
```

Comment:
We don't require parameter server or parameter store for inference purpose

Reply:
These classes not required mentioned above are removed.